Resfeber/Views/WaterfallLayout.swift

```diff
@@ -104,10 +104,10 @@ class WaterfallLayout: UICollectionViewLayout {
         }
         return visibleLayoutAttributes
     }
-    
+
     override func layoutAttributesForItem(at indexPath: IndexPath)
-    -> UICollectionViewLayoutAttributes? {
+        -> UICollectionViewLayoutAttributes?
+    {
```

Comment:
I would prefer to see the return and opening brace on the same line as the rest of function declaration. Thoughts?